src/queries/next-resource.sql

```diff
@@ -1,8 +1,9 @@
 SELECT req_url, count(req_http_X_CDN_Request_ID) AS visits, resp_http_Content_Type, status_code 
+FROM ^tablename
 WHERE 
-  # resp_http_Content_Type LIKE "text/html%" AND
+  resp_http_Content_Type LIKE "text/html%" AND
   status_code LIKE "404"
```

Comment:
Since you're doing an exact match here (e.g. not a wildcard) I would use `=` here:

`status_code = "404"`

Reply:
This is more on the query side; I'll make an issue for this.

Reply:
Got it;